testing/tests/DevExpress.ui.widgets/multiView.tests.js

```diff
@@ -656,6 +733,19 @@ QUnit.module('loop', {
         assert.equal(multiView.option('selectedIndex'), 2, 'selected index changed correctly');
     });
 
+    QUnit.test('selected index should be set to second item after right swipe from first one if last is disabled', function(assert) {
```

Comment:
Why do we need this test?
I believe that "selected index should be set to second item after right swipe from first one" even if last item is not disabled.
The same question to a test below

Reply:
When we swipe right, the container moves to the left. Accordingly, in the loop mode, we will move from 1 to 3 item if 3 is available. If 3 is not available, then we must move from 1 to 2.

The same answer to a test below, but we swipe left and the container moves to the right.